til-domain/src/main/java/com/til/application/interview/InterviewService.java

```diff
@@ -17,10 +20,15 @@ public class InterviewService {
 
     private final InterviewRepository interviewRepository;
 
+    private final RandomIdProvider randomIdProvider;
+
+    private static final int RANDOM_ID_SIZE = 11;
+
     @Transactional
     public InterviewCodeDto create(InterviewCreateDto interviewCreateDto) {
-        // todo: URL에 사용할 짧고 고유한 code 생성하여 대체
-        String code = "fresh code";
+
+        String code = randomIdProvider.provideRandomId(RANDOM_ID_SIZE);
+        checkCode(code);
```

Comment:
P4: `createRandomId()`같은 메서드를 정의해서 아래와 같이 변경해봐도 좋을 것 같습니다!
```suggestion
        String code = createRandomId();
        ...
}

private String createRandomId() {
        String code = randomIdProvider.provideRandomId(RANDOM_ID_SIZE);
        checkCode(code);
        return code;
}
```

Reply:
이 부분은 반영해도 괜찮을 것 같습니다!